quickwit-actors/src/async_actor.rs

```diff
@@ -131,13 +130,17 @@ async fn process_msg<A: Actor + AsyncActor>(
     }
 
     match command_or_msg_recv_res {
-        Ok(CommandOrMessage::Command(cmd)) => process_command(actor, cmd, ctx, state_tx),
+        Ok(CommandOrMessage::Command(cmd)) => {
+            ctx.start_processing();
+            process_command(actor, cmd, ctx, state_tx)
+        }
         Ok(CommandOrMessage::Message(msg)) => {
             let span = actor.message_span(msg_id, &msg);
```

Comment:
if your first message is message then I believe this is wrong.

Reply:
oh yes totally wrong, I was not really awake when I committed this.